m01_introduction/01_gallery.rst

```diff
@@ -0,0 +1,158 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
```

Comment:
This include is not needed here if you use the conf.py like I said in the comment inside the global.rst file.